third_party/repositories/scala_3_3.bzl

```diff
@@ -0,0 +1,536 @@
+scala_version = "3.3.0"
+scalatest_version = "3.2.16"
+netty_version = "4.1.32.Final"
+grpc_version = "1.24.0"
+specs2_version = "5.0.0-RC-21"
```

Comment:
Could you please inline version vars, except `scala_version` which is referred in other macros?  I'd like to keep this file easy to grep

Reply:
o7 np, thought i was being clever